docs/tutorials/geomesa-quickstart-fsds.rst

```diff
@@ -171,12 +171,62 @@ contained in ``org.geomesa.example.data.GDELTData``:
 -  ``getTestData`` parses an embedded TSV file to create ``SimpleFeature`` objects
 -  ``getTestQueries`` illustrates several different query types, using CQL (GeoTools' Contextual Query Language)
 
-Visualize Data With GeoServer (optional)
-----------------------------------------
+Visualize Data (optional)
+-------------------------
+
+There are two options to visual the data ingested by this quick start. The easiest option is to use the
+``export`` command of the GeoMesa FSDS tools distribution. For a more production ready example, you can
+alternatively stand up a GeoServer and connect it to your FSDS instance.
+
+Visualize Data With Leaflet
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+    In order to successfully run this command you must have a computer that is connected to the internet
+    in order to access external leaflet resources.
+
+
+.. note::
+
+    If you are running GeoMesa on a headless server (no GUI) you can still run this command to generate
+    the leaflet map and then copy the ``index.html`` to another computer.
+
+
+The ``export`` command is a part of the GeoMesa FSDS command-line tools. In order to use the command,
+ensure you have the command-line tools installed as described in :ref:`setting_up_fsds_commandline`.
+The ``export`` command provides the ``leaflet`` format which will export the features to a leaflet map
+and attempt to open the resulting HTML in your default browser. To produce the map, run the following
+command from the GeoMesa FSDS tools distribution directory:
+
+.. code:: bash
+
+    bin/geomesa-fs export         \
+        --output-format leaflet   \
+        --instance <instance>     \
+        --zookeepers <zookeepers> \
+        --user <user>             \
+        --password <password>     \
+        --catalog <table>
+
+
+Where the connection parameters are the same you used above during the quickstart. If you ran the
+command on a computer with a default web browser you should see the map below automatically open in it.
```

Comment:
minor thing - could we remove the `note` above, and then change this line to be:

```
If you ran the command on a computer with a default web browser you should see the map below open automatically. If you are running on a headless server (no GUI), you can copy the ``index.html`` to another computer and open it there.
```